bin/check-cfn-change

```diff
@@ -12,27 +12,27 @@ TIMESTAMP=$(date +%s)
 REGION=us-west-2
 
 echo "checking if a changeset exists already..."
-if aws cloudformation describe-change-set --region ${REGION}  --change-set-name auto-changeset-pr-$PR_NUMBER  --stack-name civiform > /dev/null; then
+if aws cloudformation describe-change-set --region ${REGION}  --change-set-name auto-changeset-pr-$PR_NUMBER  --stack-name civiform-staging > /dev/null; then
```

Comment:
Thought(non-blocking): mind making this a variable? we might be able to reuse this script to detect changes to different environments. Feel free to ignore if not useful.

Reply:
Done, good idea.